clap_builder/src/builder/possible_value.rs

```diff
@@ -130,6 +133,20 @@ impl PossibleValue {
         self
     }
 
+    /// Add a _visible_ alias to this [PossibleValue].
+    ///
+    /// Unlike [PossibleValue::alias], these aliases will show in help output.
+    #[must_use]
+    pub fn visible_alias(mut self, name: impl IntoResettable<Str>) -> Self {
```

Comment:
Search for `aliases_command` and add cases that use this (and invisible aliases)

Again, modify the tests in a prior commit (pass) and then switch tests from regular aliases to visible aliases.